tests/test_sageintacct/test_tasks.py

```
@@ -961,6 +961,88 @@ def test_create_ap_payment(mocker, db):
+    mocker.patch('sageintacctsdk.apis.Reimbursements.post', return_value=data['reimbursements'])
+    mocker.patch('fyle_integrations_platform_connector.apis.Reimbursements.sync', return_value=None)
+    mocker.patch('sageintacctsdk.apis.Bills.update_attachment', return_value=data['bill_response'])
+    mocker.patch('fyle_integrations_platform_connector.apis.Expenses.get', return_value=[])
+
+    workspace_id = 1
+    expense_group = ExpenseGroup.objects.get(id=1)
+    task_log = TaskLog.objects.filter(expense_group__workspace_id=workspace_id).first()
+    task_log.status = 'READY'
+    task_log.save()
+    
+    create_bill(expense_group, task_log.id, True)
```

Ruff

`create_bill` may be undefined, or defined from star imports

(F405)